flight/Modules/OnScreenDisplay/onscreendisplay.c

```diff
@@ -731,14 +731,18 @@ void draw_alarms(int x, int y, int xs, int ys, int va, int ha, int flags, int fo
 		buf[strlen(boot_reason)] = '\0';
```

Comment:
Should only copy max of sizeof(buf)-1 in the above or this could write past the end of the buffer by 1 byte.